xml/MAIN.susestorage.xml

```diff
@@ -23,8 +23,9 @@
         <dm:languages/>
         <dm:release>SES 6</dm:release>
         <dm:bugtracker>
-          <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
-          <dm:component>Documentation</dm:component>
+          <dm:url>https://github.com/SUSE/doc-ses/issues/new</dm:url>
+          <dm:assignee>asettle</dm:assignee>
+          <dm:labels>bug,question</dm:labels>
           <dm:product>&productname; &productnumber;</dm:product>
           <dm:assignee>tbazant@suse.com</dm:assignee>
```

Comment:
These two don't work with GitHub.
dm:product is unsupportted, dm:assignee must be a GitHub user name,

```suggestion
          <dm:assignee>tbazant</dm:assignee>
```